unified-codes/apps/data-service/src/app/api.ts

```diff
@@ -11,12 +11,13 @@ import { KeyCloakIdentityProvider } from './IdentityProvider';
 export const createApolloServer = (_typeDefs, _resolvers, _dataSources, _authenticator) => {
   const AUTH_URL = 'http://127.0.0.1:9990/auth/realms/unified-codes';
 
+  const getDataSources = () => ({
```

Comment:
ignore previous comment. the reason that I can't do as you ask is that the `dataSources` has to be a method not an object. Using `??` with the inline method is what fails.

Reply:
Ah ok! Fair enough, then!